sdk/servicebus/azure-servicebus/azure/servicebus/_common/receiver_mixins.py

```diff
@@ -50,12 +50,11 @@ def _populate_attributes(self, **kwargs):
         self._last_received_sequenced_number = None
         self._message_iter = None
         self._connection = kwargs.get("connection")
-        prefetch_count = kwargs.get("prefetch_count", 0)
-        if int(prefetch_count) < 0 or int(prefetch_count) > 50000:
+        self._prefetch_count = kwargs.get("prefetch_count", 0)
```

Comment:
do we want to cast as int here itself?
```suggestion
        self._prefetch_count = int(kwargs.get("prefetch_count", 0))
```